<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiceBookMarkWidget</class>
 <widget class="QWidget" name="DiceBookMarkWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DiceShortCut</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableView" name="tableView">
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragOnly</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::CopyAction</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QToolButton" name="m_addBtn">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../../../rolisteam.qrc">
           <normaloff>:/resources/icons/add_round.png</normaloff>:/resources/icons/add_round.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_removeBtn">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../../../rolisteam.qrc">
           <normaloff>:/resources/icons/delete.png</normaloff>:/resources/icons/delete.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../rolisteam.qrc"/>
  <include location="../../../../rolisteam.qrc"/>
 </resources>
 <connections/>
</ui>
